test/projection_test.py

```diff
@@ -1,23 +1,51 @@
 #!/usr/bin/env python3
+# Copyright (c) 2024, Open Source Robotics Foundation, Inc.
+# All rights reserved.
+# Redistribution and use in source and binary forms, with or without
```

Comment:
This should likely be:

```suggestion
# Copyright (c) 2014, Enrique Fernandez
# All rights reserved.

# Redistribution and use in source and binary forms, with or without
```

(it came in with 0ed1dafb145039ff58cc6b141893713dc918d867, around the same time as the rest)